config/routes.rb

```diff
@@ -12,11 +12,13 @@
     resources :invoice_items, only: [:update]
   end
 
+  get "/merchants/:merchant_id/items/new", to: "items#new"
   get "/merchants/:merchant_id/items/:item_id", to: "items#show"
   get "/merchants/:merchant_id/items/:item_id/edit", to: "items#edit"
   patch "/merchants/:merchant_id/items/:item_id", to: "items#update"
   get "/merchants/:merchant_id/invoices", to: "invoices#index"
   get "/merchants/:merchant_id/invoices/:invoice_id", to: "invoices#show"
```

Comment:
We might be able to get rid of some of these routes as invoices index and invoices show is in our resources we also might be able to do it for all these item paths